### DISCUSSION TITLE
Github Copilot - [Extension Activation Failed]

### DISCUSSION THREAD

**Discussion by mateusloubach** [community/community#107456] [Copilot Conversations] [answered]
So, yes. - I have read many other discussions here in order to find a solution to this issue that has appeared from time to time, but I can't see to fix this, even with the solutions people have provided in those other discussions.

I actually believe this is a new issue because it worked fine the day before I applied for the Github Campus Expert Program. And since we need to update some information in the Github platform, I believe this caused it. I could be wrong because I'm very noob.

These are the messages that appeared:

When I click "enable" to activate the extension in VS CODE, this appears:
"Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as mateusloubach."

then, when I click to read log, this is what appears:

[info] [auth] Invalid copilot token: missing token: 403 
[error] [default] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as mateusloubach"

Thanks in advanced.

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by mateusloubach** [reply]
This is the same answer I've seen in those other posts. Tried again and still did not fix issue.

**Reply by davecheney** [reply]
Thank you. I have confirmed that @mateusloubach does not currently have a copilot subscription. This is why copilot is not available for you.

**Reply by mateusloubach** [reply]
But I was using it yesterday normally. How come I no longer have a subscription? Did changing my billing information cancel my subscription?

**Reply by davecheney** [reply, marked as answer]
We cannot provide billing support in this forum, please contact support https://support.github.com/contact
